Pizzeria.AppHost/Program.cs

```diff
@@ -0,0 +1,29 @@
+using Aspire.Hosting;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+// Add infrastructure services
+var postgres = builder.AddPostgres("postgres")
+    .WithPgAdmin()
+    .AddDatabase("pizzeria");
+
+var rabbitmq = builder.AddRabbitMQ("rabbitmq")
+    .WithManagementPlugin();
+
+var seq = builder.AddSeq("seq");
+
+// Add application services
+var store = builder.AddProject<Projects.Pizzeria_Store>("pizzeria-store")
+    .WithReference(postgres)
+    .WithReference(rabbitmq)
+    .WithReference(seq);
+
+builder.AddProject<Projects.Pizzeria_Kitchen>("pizzeria-kitchen")
+    .WithReference(rabbitmq)
+    .WithReference(seq);
+
+builder.AddProject<Projects.Pizzeria_Delivery>("pizzeria-delivery")
+    .WithReference(rabbitmq)
+    .WithReference(seq);

```

Comment:
Create a new C# project called `Pizzeria.Common` and add a `ServiceNames` class similar to [this](https://github.com/TheMagnificent11/lewee/blob/e83735bcb5a287d1cfabb6b9b0a832a6e28bc05b/sample/Pizzeria.Common/ServiceNames.cs#L3).

Add a reference to the new `Pizzeria.Common` project with the `IsAspireProjectResource` attribute set to `false` (see [here](https://github.com/TheMagnificent11/lewee/blob/e83735bcb5a287d1cfabb6b9b0a832a6e28bc05b/sample/Pizzeria.AppHost/Pizzeria.AppHost.csproj#L17) for an example).

Don't reference magic strings in other projects, instead reference the constants in `ServiceNames`.